### PR TITLE
MWPW-120932: Org Logo JSON LD Schema

### PR DESCRIPTION
**Pull request by fullcolorcoder**
Adds organization as a type for JSON LD data in the metadata table. 

Resolves: [MWPW-120932](https://jira.corp.adobe.com/browse/MWPW-120932)

Rich Results Test: https://search.google.com/test/rich-results/result?id=V6Va-r_fW4q6sphEqj5ngA

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/bradjohn/org-logo-schema?martech=off
- After: https://org-schema--milo--adobecom.hlx.page/drafts/bradjohn/org-logo-schema?martech=off

### CODE REVIEW

File: libs/features/richresults.js [hunk 1]
```diff
@@ -33,12 +33,23 @@ function getRichResultsForSiteSearchBox(getMetadata) {
...
+  };
+}
+
 function getRichResults(type, getMetadata) {
   switch (type) {
     case 'NewsArticle':
       return getRichResultsForNewsArticle(getMetadata);
     case 'SiteSearchBox':
       return getRichResultsForSiteSearchBox(getMetadata);
+    case 'organization':
```
Comment (hparra) [inline libs/features/richresults.js:51]: nit: Uppercase to match Schema.org names but I realize this is passed in from `<meta name="richresults" content="organization">` which may already exist.

Comment (fullcolorcoder) [reply libs/features/richresults.js]: Yea I agree. Word will auto capitalize this too it seems, so good call thanks :)

**Review by jckautzmann — APPROVED** [review]

**Comment by fullcolorcoder** [comment]
@amauch-adobe I assigned this to you quick, if you have the bandwidth would you mind verifying for me?

**Comment by hparra** [comment]
Related unit test is failing

**Review by hparra — COMMENTED** [review]
Related Organization unit test is failing.
Is the case being changed case somewhere to reflect capitalization in switch?

**Review by amauch-adobe — CHANGES_REQUESTED** [review]
Everything looks good with the test urls on all browsers, there were no console errors, and I found the JSON schema on the test pages.  The only concern I have is with what everyone else has said about the unit test failure.  Once that is working I'm okay with approving.

**Comment by fullcolorcoder** [comment]
@amauch-adobe should be good now

**Review by amauch-adobe — COMMENTED** [review]
Approved.

**Comment by hparra** [comment]
We changed test. There is no indication that `getMetadata` normalizes casing and I think metadata block lives elsewhere (franklin?). Are we sure if someone uses lowercase "organization" in metadata block that switch case with catch? I understand this problem exists for the other cases as well.

**Pull request merged**